app/src/main/java/org/developfreedom/logmein/DatabaseEngine.java

```diff
@@ -106,8 +106,8 @@ public boolean insert(UserStructure us){
         } catch (Exception e) {
```

Comment:
Potential missing `cursor.close()` method

```suggestion
        cursor.close();
        } catch (Exception e) {
```

Reply:
fixrbot patterns

Reply:
API Methods called in the pattern:

```
Cursor.MoveToFirst()
Cursor.getString()
Cursor.close()
```
Pattern popularity: 94
Edit Distance: 3

Reply:
fixrbot example

Reply:
> fixrbot example

**asantibanez/Patio/**[src/main/java/com/andressantibanez/android/patio/PatioUtils.java:getRealPathFromURI](https://github.com/asantibanez/Patio/blob/2596a912c7302db7e86a8ae13aeb3e8958ce8532/Patio/src/main/java/com/andressantibanez/android/patio/PatioUtils.java#L17-L27)